app/views/unattended/provisioning_templates/snippet/pxegrub_chainload.erb

```diff
@@ -3,7 +3,9 @@ kind: snippet
 name: pxegrub_chainload
 model: ProvisioningTemplate
 snippet: true
-%>
+description: |
+  TODO
```

Comment:
@lzap could you please provide a suggestion?

Reply:
Snippet used to chainload BIOS HDD when booted from network. Used in PXEGrub global and local templates.